backbone.js

```diff
@@ -1647,14 +1647,15 @@
       // support the `hashchange` event, HTML5 history, or the user wants
       // `hashChange` but not `pushState`.
       if (!this._hasHashChange && this._wantsHashChange && !this._usePushState) {
-        var iframe = document.createElement('iframe');
-        iframe.src = 'javascript:0';
-        iframe.style.display = 'none';
-        iframe.tabIndex = -1;
+        this.iframeElement = document.createElement('iframe');
+        this.iframeElement.src = 'javascript:0';
+        this.iframeElement.style.display = 'none';
+        this.iframeElement.tabIndex = -1;
         var body = document.body;
         // Using `appendChild` will throw on IE < 9 if the document is not ready.
-        this.iframe = body.insertBefore(iframe, body.firstChild).contentWindow;
-        this.iframe.document.open().close();
+        this.iframe = body.insertBefore(this.iframeElement, body.firstChild).contentWindow;
```

Comment:
Instead of storing both `iframeElement` and `iframe`, how about just using just `iframe` (which would be the element, not the `contentWindow`). We can use a local variable to reference the `contentWindow` when we're `open`/`closing` it.


```js
this.iframe = document.createElement('iframe');
//...
var iWindow = this.iframe.contentWindow;
iWindow.document.open();
iWindow.document.close();
```